sdk/dotnet/AlertConfiguration.cs

```diff
@@ -13,6 +13,62 @@ namespace Pulumi.Mongodbatlas
     /// `mongodbatlas..AlertConfiguration` provides an Alert Configuration resource to define the conditions that trigger an alert and the methods of notification within a MongoDB Atlas project.
     /// 
     /// &gt; **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+    /// 
+    /// ## Example Usage
+    /// 
+    /// 
+    /// 
+    /// ```csharp
+    /// using Pulumi;
+    /// using Mongodbatlas = Pulumi.Mongodbatlas;
```

Comment:
Hmm, I missed that we have this unfortunate casing for `Mongodbatlas`... But that's not a problem of this PR, it's the same in code. Maybe we'll fix it one day with a major version upgrade.

Reply:
We can review these namespaces :/